hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/ContainerEndpoint.java

```diff
@@ -573,9 +640,14 @@ public Response getContainerMisMatchInsights() {
       throw new WebApplicationException(ex,
           Response.Status.INTERNAL_SERVER_ERROR);
     }
-    return Response.ok(containerDiscrepancyInfoList).build();
+    Map<String, Object> response = new HashMap<>();
+    response.put("prevKey", prevKey);
+    response.put("missingContainerList", containerDiscrepancyInfoList);
```

Comment:
Don't name it as "missingContainerList". We can keep as "containerDiscrepancyInfo" only

Reply:
Done!